--- a/pcmanfm/pcmanfm.conf.in
+++ b/pcmanfm/pcmanfm.conf.in
@@ -8,15 +8,21 @@ autorun=1
 
 [desktop]
 wallpaper_mode=crop
-wallpaper=@prefix@/share/lxde/wallpapers/lxde_blue.jpg
+wallpaper=@prefix@/share/mdk/backgrounds/default.jpg
 desktop_bg=#000000
 desktop_fg=#ffffff
 desktop_shadow=#000000
+desktop_font=Droid Sans 10
 show_wm_menu=0
 
 [ui]
+always_show_tabs=0
+hide_close_btn=0
+max_tab_chars=32
 win_width=640
 win_height=480
+splitter_pos=150
+side_pane_mode=1
 side_pane_mode=places
 view_mode=icon
 show_hidden=0
